src/UpdateService.php

```diff
@@ -316,6 +316,22 @@ public function update8805() {
     return 'The API Doc deprecated entity type has been removed from the system.';
   }
 
+  /**
+   * Recreate API Doc entity definition.
+   *
+   * Rollback update8805().
+   */
+  public function update8806() {
+    \Drupal::entityTypeManager()->clearCachedDefinitions();
+    $entity_definition_update_manager = \Drupal::entityDefinitionUpdateManager();
+    $entity_definition_update_manager->installEntityType(\Drupal::entityTypeManager()->getDefinition('apidoc'));
```

Comment:
Do we need to check if the EntityType is installed (table exists)?  If someone is migrating from version 8805 to 8806 ?

Reply:
Just tested this scenario from 8805 with an existing table to 8806, and it does not throw nor log an error, so it should be good.